style.css

```diff
@@ -0,0 +1,287 @@
+@import url("https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@500&family=Roboto+Mono:wght@500&family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgba(200, 200, 200, 0.1);
+  font-family: "Roboto", Arial, “Sans Serif”, Noto, “YT Sans”, “Roboto Mono”;
+}
+
+.icon-spacing {
+  padding: 0px 10px;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.user-avatar {
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.grid-container {
+  position: relative;
+  left: 250px;
+}
+
+nav {
+  align-items: center;
+  position: fixed;
+  top: 0%;
+  width: 100%;
+  height: 50px;
+  background-color: #ffffff;
+  justify-content: space-between;
+  z-index: 1;
+}
+
+#left-navbar {
+  justify-content: space-evenly;
+  width: 12%;
+  align-items: center;
+  margin-left: 15px;
+}
+
+#mid-navbar {
+  align-self: center;
+  margin-right: 20px;
+  width: 50%;
+  justify-content: center;
+}
+
+#search-bar {
+  background-color: white;
+  width: 85%;
+  padding: 3px 10px;
+  border: 1px solid silver;
+  border-radius: 3px 0px 0px 3px;
+  font-size: medium;
+}
+
+#search-button {
+  border: 1px solid gray;
+  padding: 0px 30px;
+  border-radius: 0px 3px 3px 0px;
+}
+
+#right-navbar {
+  align-items: center;
+  width: 120px;
+  justify-content: space-between;
+  margin-right: 50px;
+}
+
+.main-video {
+  margin-left: initial;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.flex-horizontal {
+  flex-direction: row;
+}
+
+.upcoming {
+  font-family: Roboto, Arial, “Sans Serif”, Noto, “YT Sans”, “Roboto Mono”;
+  font-size: 16px;
+  color: #030303;
+}
+.input {
+  width: 550px;
+  position: relative;
+  left: 219px;
+}
+
+.container {
+  position: relative;
+  top: 85px;
+  left: 22px;
+}
+.main-fc {
+  width: 60%;
+  margin-left: 30px;
+}
+
+.other-videos {
+  width: 30%;
+}
+
+#autobutton {
+  border: none;
+  background-color: transparent;
+}
+.sidebar {
+  position: relative;
+  right: 35px;
+  top: 80px;
+}
+.image {
+  float: left;
+  height: 10px;
+  position: relative;
+  width: 168;
+  height: 94;
```

Comment:
Remember width and height are units. This being the case you need to add a unit after the number. Example use px for pixels.